docs/releases.md

```diff
@@ -4,23 +4,71 @@
 
 ### Overview
 
-Welcome to **April 2021 release** :octicons-heart-16: of **DietPi**. This release is focused to fine tune existent DietPi tools, making them easier to use, with more options.  
+Welcome to **April 2021 release** :octicons-heart-16: of **DietPi**. We have a number of updates in this version that we hope you will like. Some of the key highlights include:
 
-![DietPi version 7](assets/images/dietpi-version7.1.jpg){: width="300" }
+!!! hint "Highlights"
+    
+    - [Merged software list](#select-software-from-a-merged-list) - we simplified the software selection, using a single list ! You have now more options to choose from ! The list contains over 190 software options !
+    - [New software](#new-optimised-software-in-71) - IPFS Nodes, Common UNIX Printing System (CUPS), and more in programming (Go language, and an excellent IDE: VSCodium)
+    - [DietPi tools: New additions & notable updates](#dietpi-tools-in-71-new-notable-updates) - DietPi supports more VPN clients and it has more Dynamic DNS options. Enabling SSL/TLS certificates is easier now using the updates brought to Let's Encrypt tool.
+    - [Improvements](#changes-improvements-optimisations-in-71) - More improvments in DietPi tools (to name just a few: DietPi-LetsEncrypt, DietPi-Drive_Manager, DietPi-Config etc.) have been updated, along with more changes to more software packages.
+    - [Bug fixes](#bug-fixes) & updates to [supported SBC](#supported-sbc-updates-in-71) list
 
-[_Source: Pexels.com_](https://www.pexels.com/photo/abstract-aluminum-architectural-architecture-210158)
+Many thanks to all contributors and beta testers :octicons-heart-16: :octicons-heart-16: !! With your help we could achieve all of these updates ! Install or upgrade to this latest update - it has more things we hope you would enjoy ! Share your feedback on [Twitter](https://twitter.com/DietPi_), [DietPi Forum](https://dietpi.com/phpbb/viewforum.php?f=9&sid=e6fcb5de3776c2d8eb1b22cc1fb24ca5), and report any issue to [GitHub](https://github.com/MichaIng/DietPi/issues) !
 
-### Merged software list
+As a new thing, we already scheduled the next update. DietPi 7.2 will be launched on 29 May !
```

Comment:
Should we really do this exact statement? I mean what happen if we have a delay? We could keep this date internal, to avoid to create expectation we could not fulfil. There is a reason why RPi devs never populate a release date. It's done when it is done 😉

Reply:
My idea indeed was to make it fix from now on, so that we and users can plan with it. Aiming a date/range internally and not fulfilling it, was what we/me did mostly until now 😅, but I think it has quite a value to have reliable release schedules. Fixing the date more publicly helps to assure that it will be kept. First beta will start exactly one ween before, on Mai 22, so all larger changes that have been finished until then, are part of the next release, everything else will go into the next. That makes more sense than delaying all changes by a month to get the second half into the same release 😄.

Even if there is something urgent not making it until the date, I see no issue in pushing a small hotfix release a week later or so. The smaller the releases, the less changes/issues are accumulated, hence the smoother updates theoretically will be. E.g. a major part of all bug reports we receive is due to accumulated APT upgrades being all done at once together with the DietPi update.

Reply:
I agree, quite a large amount of issues is related to `apt` failure. But this will not go away with shorter release cycle. We already inform user about possible `apt` updates. As well, not every user will login that often. Means they will have the same amount of `apt` updates still before execution of DietPi update. We have users not login for ages or even running Stretch. Based on statistics, more than 20% running v6.33 still. 😉

Reply:
A shorter release schedules still implies the chance that users update earlier, of course it does not guarantee it. More regular APT upgrades can prevent failures with it, e.g. when multiple upgrades, especially also in combination with 3rd party repos, collide in a way. Also the chance that unavoidable APT errors are split means a chance that individual errors can be debugged easier compared to multiple individual errors that appear together or even affect each other are coming together.

The daily APT check indeed helps to prevent accumulated updates, but some may disable it while having DietPi update notification enabled. Jep it does not guarantee anything for individual cases but the chances for more regular updates done by users are undoubtedly better when updates are available more regular, which is enough for me to accept it as pro argument 😄. The other thing is broken or weakened install options or features due to upstream/package changes and allowing us and users to better plan implementation/system update schedules.

Reply:
Don't get me wrong. I don't argue against it. Just tried to look from all angles into it.